spectral_cube/tests/test_casafuncs.py

```diff
@@ -73,19 +117,45 @@ def test_casa_mask():
 @pytest.mark.skipif(not casaOK, reason='CASA tests must be run in a CASA environment.')
 def test_casa_mask_append():
 
-    cube = SpectralCube.read('adv.fits')
+    cube = SpectralCube.read(path('adv.fits'))
 
     mask_array = np.array([[True, False], [False, False], [True, True]])
     bool_mask = BooleanArrayMask(mask=mask_array, wcs=cube._wcs,
                                  shape=cube.shape)
     cube = cube.with_mask(bool_mask)
 
-    make_casa_testimage('adv.fits', 'casa.image')
+    make_casa_testimage(path('adv.fits'), path('casa.image'))
 
-    if os.path.exists('casa.mask'):
-        os.system('rm -rf casa.mask')
+    if os.path.exists(path('casa.mask')):
+        os.system('rm -rf {0}'.format(path('casa.mask')))
+
+    maskpath = os.path.join(path('casa.image'),'casa.mask')
+    if os.path.exists(maskpath):
+        os.system('rm -rf {0}'.format(maskpath))
+
+    # in this case, casa.mask is the name of the mask, not its path
+    make_casa_mask(cube, path('casa.mask'), append_to_image=True,
+                   img=path('casa.image'), add_stokes=False,
+                   overwrite=True)
+
+    assert os.path.exists(path('casa.image/casa.mask'))
+
```

Comment:
Delete `casa.mask` and `casa.image` here. Unless they are needed in another test?